src/injected/cssSelectorEngine.ts

```diff
@@ -87,73 +87,99 @@ export function createCSSEngine(shadow: boolean): SelectorEngine {
       //   return simple;
       // if (!shadow)
       //   return;
-      const parts = split(selector);
-      if (!parts.length)
+      const selectors = split(selector).filter(parts => !!parts.length).map(parts => parts.reverse());
```

Comment:
nit: this line looks like it goes inside `split` to me.

Reply:
Done.